packages/components/src/buttons/src/Button.module.css

```diff
@@ -0,0 +1,277 @@
+    --hop-button-component-border-radius: var(--hop-shape-rounded-md);
+    --hop-button-component-focus-ring-color: var(--hop-primary-surface-hover); /* TODO: place to centralize tokens like this? */
+    --hop-button-component-md-height: 2.5rem;
+    --hop-button-component-sm-height: 2rem;
+
+    /* Primary */
+    --hop-button-component-primary-color: var(--hop-primary-text-strong);
+    --hop-button-component-primary-color-hover: var(--hop-primary-text-strong-hover);
+    --hop-button-component-primary-color-pressed: var(--hop-primary-text-active);
+    --hop-button-component-primary-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-primary-background-color: var(--hop-primary-surface-strong);
+    --hop-button-component-primary-background-color-hover: var(--hop-primary-surface-hover);
+    --hop-button-component-primary-background-color-pressed: var(--hop-primary-surface-active);
+    --hop-button-component-primary-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-button-component-primary-border: 0;
+    --hop-button-component-primary-border-hover: 0;
+    --hop-button-component-primary-border-pressed: 0;
+    --hop-button-component-primary-border-disabled: 0;
+
+    /* Secondary */
+    --hop-button-component-secondary-color: var(--hop-neutral-text);
+    --hop-button-component-secondary-color-hover: var(--hop-neutral-text-hover);
+    --hop-button-component-secondary-color-pressed: var(--hop-neutral-text-active);
+    --hop-button-component-secondary-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-secondary-background-color: var(--hop-neutral-surface);
+    --hop-button-component-secondary-background-color-hover: var(--hop-neutral-surface-hover);
+    --hop-button-component-secondary-background-color-pressed: var(--hop-neutral-surface-active);
+    --hop-button-component-secondary-background-color-disabled: var(--hop-neutral-surface-disabled);
+    --hop-button-component-secondary-border: 0.0625rem solid var(--hop-neutral-border-strong);
+    --hop-button-component-secondary-border-hover: 0.0625rem solid var(--hop-neutral-border-strong);
+    --hop-button-component-secondary-border-pressed: 0.0625rem solid var(--hop-neutral-border-active);
+    --hop-button-component-secondary-border-disabled: 0;
+
+    /* Tertiary */
+    --hop-button-component-tertiary-color: var(--hop-neutral-text);
+    --hop-button-component-tertiary-color-hover: var(--hop-neutral-text-hover);
+    --hop-button-component-tertiary-color-pressed: var(--hop-neutral-text);
+    --hop-button-component-tertiary-color-disabled: var(--hop-neutral-text-disabled);
+    --hop-button-component-tertiary-background-color: var(--hop-neutral-surface);
+    --hop-button-component-tertiary-background-color-hover: var(--hop-neutral-surface-hover);
+    --hop-button-component-tertiary-background-color-pressed: var(--hop-neutral-surface-weak-active);
```

Comment:
This is odd, this should, in theory, use neutral-surface-active as the base color is not weak

Reply:
I verified and yes it's not supposed to be weak.
I see we decided to go with the active state being the pressed state. but Igloo also has an active prop that will show this state.
For example, if you use a button to open an action menu, we want the button to have the active state look when the menu is open. This has been used in OV.
Which means we will probably need to add an active class. Unless there's a different way we can do this?

Reply:
We'll create a separate component called ToggleButton for the button forcing an active state
- https://wl-orbiter-website.netlify.app/?path=/docs/button--default-story#toggle-button
- https://react-spectrum.adobe.com/react-aria/ToggleButton.html
- https://react-spectrum.adobe.com/react-spectrum/ToggleButton.html

Reply:
Right I forgot about that, but wouldn't it be using a lot of duplicate code and css? or will we use The Button component inside ToggleButton?

Reply:
this is one of the main issue i still need to figure out with buttons. should we have a base button, or reuse styles, or reuse component tokens?